src/stats_section/rankingTable.test.js

```diff
@@ -0,0 +1,86 @@
+const createRankingObject = history => {
```

Comment:
you should put these functions in a rankingTable.js file and export them
then you import in test files and in your componenet file

Reply:
fixed